src/SprykerEco/Zed/Heidelpay/Dependency/Injector/OmsDependencyInjector.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Heidelpay\Dependency\Injector;
+
+use Spryker\Zed\Kernel\Container;
+use Spryker\Zed\Kernel\Dependency\Injector\AbstractDependencyInjector;
+use Spryker\Zed\Oms\Communication\Plugin\Oms\Command\CommandCollectionInterface;
+use Spryker\Zed\Oms\Communication\Plugin\Oms\Condition\ConditionCollectionInterface;
+use Spryker\Zed\Oms\OmsDependencyProvider;
+use SprykerEco\Zed\Heidelpay\Communication\Plugin\Checkout\Oms\Command\AuthorizePlugin;
+use SprykerEco\Zed\Heidelpay\Communication\Plugin\Checkout\Oms\Command\CapturePlugin;
+use SprykerEco\Zed\Heidelpay\Communication\Plugin\Checkout\Oms\Command\DebitPlugin;
+use SprykerEco\Zed\Heidelpay\Communication\Plugin\Checkout\Oms\Condition\IsAuthorizationCompletedPlugin;
+use SprykerEco\Zed\Heidelpay\Communication\Plugin\Checkout\Oms\Condition\IsCaptureApprovedPlugin;
+use SprykerEco\Zed\Heidelpay\Communication\Plugin\Checkout\Oms\Condition\IsDebitCompletedPlugin;
+
+class OmsDependencyInjector extends AbstractDependencyInjector
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    public function injectBusinessLayerDependencies(Container $container)
+    {
+        $container = $this->injectCommands($container);
```

Comment:
I thought we are not using them anymore?

Reply:
yes, you are right, @mykyta65536 please remove them.